src/Core/src/Platform/iOS/ViewExtensions.cs

```diff
@@ -410,7 +410,14 @@ public static async Task UpdateBackgroundImageSourceAsync(this UIView platformVi
 				if (backgroundImage == null)
 					return;
 
-				platformView.BackgroundColor = UIColor.FromPatternImage(backgroundImage);
+				platformView.RemoveBackgroundLayer();
+				var backgroundLayer = new CALayer
+				{
+					Name = BackgroundLayerName,
+					Contents = backgroundImage.CGImage,
+					Frame = platformView.Frame,	
```

Comment:
There is no specification of how image should fit the layer. Can manage it using `ContentsGravity` and `MasksToBounds` properties.